### PR TITLE
Rework word-count documentation

### PR DESCRIPTION
**Pull request by kytrinyx**
**This exercise is one of the focus exercises for 12in23.**

This PR is part of our project of making our Practice Exercises more consistent and human. For more context please [see this forum thread](https://forum.exercism.org/t/new-project-making-practice-exercises-more-consistent-and-human-across-exercism/3943)

We aim to merge this PR in roughly 72 hours, and then submit PRs to all tracks that have this exercise.

The main change is to frame the exercise within the context of a story.

### CODE REVIEW

File: exercises/word-count/instructions.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+Your task is to count how many times each word occurs in a subtitle of a drama.
+
+The dramas that the subtitles are taken from use only ASCII characters.
```
Comment (IsaacG) [inline exercises/word-count/instructions.md:5]:
```suggestion
The subtitles from these dramas use only ASCII characters.
```

File: exercises/word-count/instructions.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The characters often speak in casual English, using contractions like _they're_ or _it's_.
+Though these contractions come from two words (e.g. _we are_), the contraction (_we're_) is considered a single word.
+
+Words can be separated by any form of punctuation (e.g. ":", "!", or "?") or whitespace (e.g. "\t", "\n", or " ").
+The only punctuation that does not separate words is the apostrophe in contractions.
+
+Numbers are considered words.
+So if the subtitles say `It costs 100 dollars.` then `100` will be its own word.
```
Comment (IsaacG) [inline exercises/word-count/instructions.md:14]:
```suggestion
If the subtitles say `It costs 100 dollars.` then `100` will be its own word.
```

File: exercises/word-count/instructions.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Words are case insensitive.
+For example, the word `you` occurs three times in the following sentence:
```
Comment (IsaacG) [inline exercises/word-count/instructions.md:17]: _they're_ uses italics on line 8. `you` uses backticks. Are these intentionally different?

Comment (kytrinyx) [reply exercises/word-count/instructions.md:17]: No, I completely missed that. Let me take another look at the whole file.

**Comment by kytrinyx** [comment]
Thanks for all those tweaks, @IsaacG!

**Review by IsaacG — APPROVED** [review]

**Review by BethanyG — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**